website/src/community/contact-us.md

```diff
@@ -29,12 +29,14 @@ whichever one seems best!
 
 | How to contact us | When to use it |
 | ----------------- | ---------------|
-| [user@](https://lists.apache.org/list.html?user@beam.apache.org) mailing list | User support and questions ([Subscribe](mailto:user-subscribe@beam.apache.org), [Unsubscribe](mailto:user-unsubscribe@beam.apache.org)) |
-| [dev@](https://lists.apache.org/list.html?dev@beam.apache.org) mailing list | Development discussions ([Subscribe](mailto:dev-subscribe@beam.apache.org), [Unsubscribe](mailto:dev-unsubscribe@beam.apache.org)) |
-| [commits@](https://lists.apache.org/list.html?commits@beam.apache.org) mailing list | Firehose of commits, bugs, pull requests, etc. ([Subscribe](mailto:commits-subscribe@beam.apache.org), [Unsubscribe](mailto:commits-unsubscribe@beam.apache.org)) |
+| [user@](https://lists.apache.org/list.html?user@beam.apache.org) mailing list | User support and questions ([Subscribe](mailto:user-subscribe@beam.apache.org)[^1], [Unsubscribe](mailto:user-unsubscribe@beam.apache.org)[^1], [Archives](https://lists.apache.org/list.html?user@beam.apache.org)) |
+| [dev@](https://lists.apache.org/list.html?dev@beam.apache.org) mailing list | Development discussions ([Subscribe](mailto:dev-subscribe@beam.apache.org)[^1], [Unsubscribe](mailto:dev-unsubscribe@beam.apache.org)[^1], [Archives](https://lists.apache.org/list.html?dev@beam.apache.org)) |
+| [commits@](https://lists.apache.org/list.html?commits@beam.apache.org)[^1] mailing list | Firehose of commits, bugs, pull requests, etc. ([Subscribe](mailto:commits-subscribe@beam.apache.org)[^1], [Unsubscribe](mailto:commits-unsubscribe@beam.apache.org)[^1], [Archives](https://lists.apache.org/list.html?commits@beam.apache.org)) |
 | [JIRA bug tracker](https://issues.apache.org/jira/browse/BEAM) | Report bugs / discover known issues |
 | [StackOverflow](http://stackoverflow.com/questions/tagged/apache-beam) | Ask and answer user support questions |
 | [Slack](https://s.apache.org/beam-slack-channel) | Chat with users and developers on channel #beam ([Join](https://s.apache.org/slack-invite)) |
 {:.table}
 
+[^1]: To subscribe or unsubscribe, a blank email is fine.
```

Comment:
This actually formats with a bunch of silly backlinks. Looks funny, but I don't know if it's worth looking into fixing.

http://apache-beam-website-pull-requests.storage.googleapis.com/6857/community/contact-us/index.html#fnref:1